### DISCUSSION TITLE
Adcirc data assimilation problem

### DISCUSSION THREAD

**Discussion by q1165600785** [adcirc/adcirc#410] [General]
May I ask if adcirc can perform data assimilation? Do I have some tidal level data from tidal stations that can be assimilated

**Comment by BrianOBlanton** [comment]
Yes, see

Asher et al., 2019] Asher, T. G., Luettich Jr, R. A., Fleming, J. G., and Blanton, B. O. (2019). Low fre- quency water level correction in storm surge models using data assimilation. Ocean Modelling, 144:101483.



--

Cheers,

 [A black text on a white background  Description automatically generated]

Brian O. Blanton, Ph.D.
Director of Earth Data Sciences
Oceanographer
Renaissance Computing Institute
University of North Carolina at Chapel Hill
100 Europa Drive
Suite 540 Chapel Hill, NC, 27517

***@***.******@***.***>
919-445-9620 (O)



From: q1165600785 ***@***.***>
Date: Thursday, October 10, 2024 at 4:55 AM
To: adcirc/adcirc ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [adcirc/adcirc] Adcirc data assimilation problem (Discussion #410)

May I ask if adcirc can perform data assimilation? Do I have some tidal level data from tidal stations that can be assimilated

**Reply by q1165600785** [reply]
I have read your article. Can you provide the source code or experimental process files? I am a novice in ocean modeling and data assimilation. I still need a lot of reading and research to really understand the core of this paper. I really want to see the code you need to implement this article. I wonder if you can provide it.

**Comment by tgasher** [comment]
For basics on inputting corrections into the ADCIRC model, see here:
https://wiki.adcirc.org/Dynamic_water_level_correction

As to the paper, the dataset, including all code, is linked to in the
article.  See the link at the end of the Background section.  If you have
further questions, please contact the author of the paper (me) via the
email by my name in the paper.
.
